MinecraftClient/Program.cs

```diff
@@ -229,12 +230,37 @@ private static void InitializeClient()
                 if (Settings.DebugMessages)
                     Translations.WriteLine("debug.session_id", session.ID);
 
-                //ProtocolHandler.RealmsListWorlds(Settings.Username, PlayerID, sessionID); //TODO REMOVE
+                List<string> availableWorlds = new List<string>();
+                if (Settings.DisplayRealmsWorld)

```

Comment:
Also skip Realms when running in offline mode? (For setting name see suggestion below)
```suggestion
                if (Settings.MinecraftRealmsEnabled && !String.IsNullOrEmpty(session.ID))
```